packages/runtime/src/lib/pycompile-op.ts

```diff
@@ -1,5 +1,5 @@
 import { BuilderError, ERRORS } from "@algo-builder/web";
-import { spawnSync, SpawnSyncReturns } from "child_process";
+import { spawnSync, SpawnSyncReturns, exec } from "child_process";
```

Comment:
where do we use it?

Reply:
In the same file inside `validatePythonModule` method.